operations/network/NMN_Isolation.md

```diff
@@ -0,0 +1,243 @@
+# Node Management Network (NMN) Isolation
+
+## NMN Isolation Overview
+
+NMN Isolation on the management network limits traffic on the NMN to only types and directions required for the operation of CSM and user workloads.
+**NMN Isolation is only available on systems with Aruba switches**. The feature consists of three main sub-features:
+
+- ACLs allowing access to only required CSM services from managed nodes (compute, UAN, and so on)
+- ACLs preventing Mountain compute cabinets (EX) from communicating with each other
+- PVLAN to prevent River managed nodes (compute, UAN, and so on) from communicating with each other
+
+NMN Isolation alleviates the need for most host-based firewalls on management nodes (managers, workers, and storage NCN).
+The PVLAN sub-feature also removes the need for host-based firewalls on UAN (and other River managed nodes) by limiting access over the NMN.
+
+NMN Isolation is available in the following commands of CANU via the `--enable-nmn-isolation --enable-pvlan` options:
+
+- `canu generate switch config` generates a configuration file for a _single switch_ on the management network
+- `canu generate network config` generates all configuration files for _all switches_ on the management network
+- `canu validate switch config` compares the running configuration of a _single switch_ with a configuration generated by CANU - a configuration diff
+- `canu validate network config` provides a summary comparison between generated and running configurations for _all switches_ in the management network
+- `canu test` runs a full diagnostic on all management network switches, including tests for NMN Isolation ACLs
+
+Details of allowed services and the network changes involved in the NMN Isolation feature can be found in [NMN Isolation Details](#nmn-isolation-details).
+
+## Enabling NMN Isolation
+
+A network outage window is required to configure NMN Isolation on a system.  NMN Isolation changes are decoupled from other changes to CSM and the network outage window could be prior or after the CSM upgrade.
+However, if IPv6 is being enabled as part of a system upgrade, enabling NMN Isolation at the same time as IPv6, and prior to the CSM upgrade is required to avoid multiple network outage windows.
```

Comment:
```suggestion
However, if IPv6 is being enabled as part of a system upgrade, enabling NMN Isolation at the same time as IPv6, and prior to the CSM upgrade, is required to avoid multiple network outage windows.
```